homeassistant/components/climate/generic_thermostat.py

```diff
@@ -41,8 +43,10 @@
 CONF_COLD_TOLERANCE = 'cold_tolerance'
 CONF_HOT_TOLERANCE = 'hot_tolerance'
 CONF_KEEP_ALIVE = 'keep_alive'
-CONF_INITIAL_OPERATION_MODE = 'initial_operation_mode'
```

Comment:
Why do you remove setting the initial operation mode? That change would make this a breaking change, which we should avoid if possible.

Reply:
I was looking at usages for that specific setting, especially with the functionality to restore state from "old_state". It really didn't make much sense to have this. Also this doesn't look like any other thermostat feature observed and looked like a deviation from standards. 

However looking at it from a breaking change perspective, it doesn't hurt to leave it on as you can just avoid using it. 

I will take guidance here if I should leave it or remove it.